Nodejs/Product/Nodejs/Nodejs.csproj

```diff
@@ -1531,6 +1530,11 @@
       <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup%3bBuiltProjectOutputGroupDependencies%3bGetCopyToOutputDirectoryItems%3bSatelliteDllsProjectOutputGroup%3b</IncludeOutputGroupsInVSIX>
       <IncludeOutputGroupsInVSIXLocalOnly>DebugSymbolsProjectOutputGroup%3b</IncludeOutputGroupsInVSIXLocalOnly>
     </ProjectReference>
+    <ProjectReference Include="..\Telemetry\Telemetry.csproj">
+      <Project>{d6fcbc60-f21e-4854-ac51-aed1a0a7c868}</Project>
+      <Name>Telemetry</Name>
+      <IncludeInVSIX>true</IncludeInVSIX>
+    </ProjectReference>

```

Comment:
This is a common dll independent o VS version. The dlls which log actual telemetry by implementing interfaces from this one are (will be) part of closed repo. That one will have #ifdef checks